_episodes/02-concepts.md

```diff
@@ -137,14 +195,31 @@ also capture the output necessary for scientific hypothesis testing or
 exploration. In this example, the volumes of subcortical structures and 
 of the different brain tissue classes are extracted and stored in a JSON
 document. A specific run of this workflow on a specific platform was 
-used to create the provnenace document and the expected outputs data. 
+used to create the provenance document and the expected outputs data. 
 When another user runs this workflow, their output can be compared to 
 the expected to output.
 
 Lesson 4 covers data flow technologies, specifically how to create
 analysis pipelines and applications and capture provenance when running 
 these pipelines. 
 
+> ## Exercise:
+>
+>  What are some advantages of using dataflow technologies?
+>
+> > ## Solution
+> >
+> > 1. Compact and structured representation of analysis
+> > 2. Can be reused with minimal changes
+> > 3. Many dataflow tools can be used across different environments.
+> > 4. The tools take care of data management.
+> > 5. Many dataflow tools support distributed execution of steps.
+> {: .solution}
+{: .challenge}
+
+
+<a name="element4" />
+
```

Comment:
Is this the right place to point difference between workflow vs dataflow technologies and points to  nidm-w as well?